partd/pandas.py

```diff
@@ -14,6 +15,10 @@
 
 dumps = partial(pickle.dumps, protocol=pickle.HIGHEST_PROTOCOL)
 
+_NO_EXTENSION = 0
```

Comment:
Not sure if it's worth making a full blown enum. This is a bit hacky but I think it works.